src/io/browser/browser-io.ts

```diff
@@ -39,13 +39,19 @@ export function makeBrowserIo(): EdgeIo {
     }),
 
     // Networking:
-    fetch(uri: string, opts?: EdgeFetchOptions): Promise<EdgeFetchResponse> {
-      return window.fetch(uri, opts)
-    },
-    async fetchCors(
+    async fetch(
       uri: string,
       opts?: EdgeFetchOptions
     ): Promise<EdgeFetchResponse> {
+      const { avoidCors = 'auto' } = opts ?? {}
+
+      if (avoidCors === 'never') {
+        return await fetchCorsProxy(uri, opts)
+      }
+      if (avoidCors === 'always') {
+        return await window.fetch(uri, opts)
+      }
```

Comment:
I think these two conditions are backwards.

Reply:
If we are "never avoiding cross-origin request sharing" then we are "allowing cross-origin request sharing" which means we are allowing sharing between our origin (null) and the server's, so we should not let CORS policies block.

If we're are "always avoiding cross-origin request sharing" then we are "never cross-origin request sharing" which means if the server doesn't share our origin (null) then it should fail.

This is the logic I understood it as. Which is why it seemed a bit confusing to read and may not be obvious to the user if you're deriving an opposite conclusion from what I did when reading the API.

Reply:
Went with the "bypass" name instead, to make it clear what is meant.